gadget.yaml

```diff
@@ -14,15 +14,31 @@ volumes:
         offset-write: mbr+92
         content:
           - image: pc-core.img
-      - name: EFI System
+      - name: Recovery
+        role: system-recovery
+        filesystem: vfat
+        # UEFI will boot the ESP partition by default first
         type: EF,C12A7328-F81F-11D2-BA4B-00A0C93EC93B
+        # max volume label length: 11 chars
+        filesystem-label: sys-recover
+        # too big? too small?
+        size: 1200M
+        content:
+          - source: grubx64.efi
+            target: EFI/boot/grubx64.efi
+          - source: shim.efi.signed
+            target: EFI/boot/bootx64.efi
+          - source: grub.cfg-recovery
+            target: EFI/ubuntu/grub.cfg
+      - name: System boot
+        type: 0B,EBD0A0A2-B9E5-4433-87C0-68B6B72699C7
         filesystem: vfat
         filesystem-label: system-boot
-        size: 50M
+        size: 100M
```

Comment:
I'm skeptical of this as a size for the partition given that we will need to store 2x vmlinuz (+ embedded initramfs) on this fs for the FDE case.